ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -328,6 +328,7 @@ measurement_descriptor:
   thermal: "Relating to the transfer of heat."
   tilt: "Degree of radial rotation."
   wetbulb: "Describes air temperature measured at 100% relative humidity (saturation)."
+  distance: "The length of the space between two points."
```

Comment:
Put in alphabetical order. This is also a measurement, not a descriptor -- it doesnt describe a measurement, it is the measurement.

Reply:
> Put in alphabetical order. This is also a measurement, not a descriptor -- it doesnt describe a measurement, it is the measurement.

Hi, 

The sub-field has already been updated in line 238 (https://github.com/google/digitalbuildings/blob/59ca7adeccf762810349927b44bc8f2bbeafa65b/ontology/yaml/resources/subfields/subfields.yaml#L238). 

This pull request can be deleted.

-Rathish